src/app/shared/label-dropdown/label-dropdown.component.html

```diff
@@ -8,6 +8,10 @@
       [ngStyle]="this.labelService.setLabelStyle(this.selectedColor, 'inline-block')"
       required
       disableOptionCentering
+      [matTooltipDisabled]="!dropdownControl.value"
+      [matTooltip]="
+        this.labelService.parseLabelDefinitionMarkdown(this.labelService.getLabelDefinition(dropdownControl.value, attributeName))
+      "
```

Comment:
Similarly, it might be unecessary to show tooltips when label definitions can already be readily viewed during selection.